### PR TITLE
show nginx for centos

### LINKED ISSUE #243: Document the Nginx plugin in Fedora/EPEL instructions
It has shipped!

### PR DESCRIPTION
**Pull request by SwartzCr**
fixes #243

### CODE REVIEW

File: _scripts/instruction-widget/install.js [hunk 1]
```diff
@@ -87,8 +87,6 @@ module.exports = function(context) {
 
       if (context.webserver == "apache") {
         context.package = "python-certbot-apache";
-      } else if (context.webserver == "nginx") {
-        context.certonly = true;
```
Comment (bmw) [inline _scripts/instruction-widget/install.js]: You should add the previous line back and change this line to:
```
context.package = "python-certbot-nginx";
```
Currently, the built instructions tell you to just install Certbot and then run `certbot --nginx`.

File: _scripts/instruction-widget/templates/getting-started/nginx.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Certbot has an Nginx plugin, which is supported on
 many platforms, and automates both obtaining and installing certs:
 </p>
-<pre>$ {{base_command}} --nginx</pre>
+<pre>$ sudo {{base_command}} --nginx</pre>
```
Comment (bmw) [inline _scripts/instruction-widget/templates/getting-started/nginx.html]: Nice catch!

**Comment by SwartzCr** [comment]
@bmw should be fixed now?

File: _scripts/instruction-widget/install.js [hunk 3]
```diff
@@ -88,7 +88,7 @@ module.exports = function(context) {
       if (context.webserver == "apache") {
         context.package = "python-certbot-apache";
       } else if (context.webserver == "nginx") {
-        context.certonly = true;
+          context.package = "python-certbot-nginx";
```
Comment (bmw) [inline _scripts/instruction-widget/install.js]: tiny nit: indentation is off

**Comment by SwartzCr** [comment]
okay fine, done

**Pull request merged**